content/OTP/Guides/OTP_Walk-through.adoc

```diff
@@ -0,0 +1,101 @@
+== OTP Walk-through
+
+=== Overview: YubiKeywith OTP
+
+Build an app that enables your users to register and authenticate with Yubico OTP (One-Time Password). This walk-through describes how to integrate the Yubico OTP protocol with your application or framework. This content is based on the https://www.yubico.com/wp-content/uploads/2012/10/YubiKey-Authentication-Module-Design-Guideline-v1.0.pdf[YubiKey Authentication Module Design Guideline].
+
+All YubiKeys support Yubico OTP, which can be used as single-factor authentication or as the second factor in a two-factor authentication scheme. Typically Yubico OTP is used in combination with a username and password.
+
+For OTP verification, you can host a verification server yourself, or use Yubico’s web service for this purpose, the YubiCloud. For this, you need to get an API key from https://upgrade.yubico.com/getapikey/[here] — it is quick, free and helps us prevent misuse of the YubiCloud.
+
+Yubico OTP requires no client software. The YubiKey ID is embedded in OTP, which allows for self-provisioning, as well as authenticating without a username.
+
+If you use the YubiCloud, supporting Yubico OTP is not much harder than supporting regular passwords.
+We have integration plugins for software such as Wordpress, phpBB and PAM as well as libraries for programming languages you can use to create your own integration.
+
+
+=== Create your Authentication Module
+
+Your authentication module defines the communications and actions between three components:
+
+* YubiKey – The user’s client device
+* Validation server – This can be either the YubiCloud or your own self-hosted validation service with the Internal YubiKey OTP Validation Service integrated.
+* Authentication Business Logic module – When one YubiKey provides access to multiple applications on a framework.
+
+
+image::placeholder-diagram.png[]
```

Comment:
Is this intended to still be a placeholder?

Reply:
I don't have a diagram, so I have removed it.